### PR TITLE
Fixed error with infinite stock getting offers

### PR DESCRIPTION
**Pull request by rogergros**
Proposed solution for https://github.com/elcodi/elcodi/issues/581

Ping @elcodi/owners

### CODE REVIEW

File: src/Elcodi/Bundle/ProductBundle/Resources/config/services.yml [hunk 1]
```diff
@@ -12,3 +12,4 @@ services:
         class: %elcodi.provider.product_collection.class%
         arguments:
             product_repository: @elcodi.repository.product
+            configuration_manager: @elcodi.manager.configuration
```
Comment (xphere) [inline src/Elcodi/Bundle/ProductBundle/Resources/config/services.yml]: I'd rather inject directly the configuration value with `@=elcodi_config('product.use_stock')`

Comment (mmoreram) [inline src/Elcodi/Bundle/ProductBundle/Resources/config/services.yml]: Already fixed in my work.

Comment (mmoreram) [inline src/Elcodi/Bundle/ProductBundle/Resources/config/services.yml]: Ups, nopes.

Please, inject the value instead of all Manager :)

**Comment by rogergros** [comment]
Fixed :)

**Pull request merged**